### ISSUE TITLE
Same table name, different connection

### ISSUE THREAD

**Issue by kai23** [balderdashy/sails-mysql#171]
Hello,

I'm having trouble with two models which have the same table name but which are linked with two differents databases.

table1.js

	module.exports = {
		connection: 'connect1', 
		tableName: 'table1',
		attributes: {
			xxx
		},
	};

table2.js

	module.exports = {
		connection: 'connect2',
		tableName: 'table1',
		
		attributes: {
			xxxx
		},
	};

I can't rename the table in the database, and don't want to use `query` for one of them. Here is the error I get : 

	table1.WHATEVER(xxxxx).exec(function(err, result) {
		ErrorService.log(err);
		res.json({
			success: true
		});
	})

	------------------------------

    Error (E_UNKNOWN) :: Encountered an unexpected error
    : ER_BAD_FIELD_ERROR: Unknown column 'table2.hash' in 'field list'

    Details:  Error: ER_BAD_FIELD_ERROR: Unknown column 'table2.hash' in 'field list'

Sails tries to load the configuration of `table2`.

Does anyone have a solution ?

**Comment by kai23** [comment]
Thanks. 

Do you have any idea on how I can get a better stacktrace ? All stacktrace only point to files in the node_modules folder, it's really hard to see where we made a bad request, for example.

**Comment by kai23** [comment]
to add something about this, it works well when we don't have any populate.

**Comment by dmarcelino** [comment]
This may be related to balderdashy/waterline#925 (https://github.com/balderdashy/waterline/issues/925#issuecomment-89614487) as there have been reported issues with `populate` across different connections.

**Comment by sailsbot** [comment]
Thanks for posting, @kai23.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
